test/unit/sync.coffee

```diff
@@ -217,6 +217,8 @@ describe "Sync", ->
 
 
     describe 'applied', ->
+        @timeout 5000
```

Comment:
By default, mocha has 2s timeout. The options file make it longer (5s), but I prefer to be explicit where tests are taking more than  2s. Here, `applied` can take 3s before calling its callback in case of errors.